keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -472,11 +441,10 @@ spec:
     # Additional annotations or labels for the
     # ResourceClaim could be specified here.
   spec:
-    resourceClassName: "acme-gpu"
-    parametersRef:
-      apiGroup: gpu.example.com
-      kind: GPURequirements
-      name: device-consumer-gpu-parameters
+    requests:
+    - name: gpu-request # could be used to select this device in a container when requesting more than one
+      deviceClassName: acme-gpu
+      deviceSelector: device.quantityAttribute["gpu.acme.example.com/memory"].isGreaterThan(quantity("2Gi"))
```

Comment:
Has it been decided yet if we will "simplify" this syntax or not?

Reply:
Updated with `device.` prefix (see https://github.com/kubernetes-sigs/wg-device-management/issues/23#issuecomment-2160044359).